src/main/fc/core.c

```diff
@@ -1130,7 +1130,7 @@ static FAST_CODE_NOINLINE void subTaskPidSubprocesses(timeUs_t currentTimeUs)
         startTime = micros();
     }
 
-#ifdef USE_MAG
+#if defined(USE_GPS) || defined(USE_MAG)
```

Comment:
I'm not sure this makes sense. The following `if (sensors(SENSOR_MAG)) {` can only be true if `USE_MAG` is defined. Adding an "or" for `USE_GPS` wouldn't accomplish anything.

Reply:
Correct. The conditional for `updateMagHold()` is for `#if defined(USE_GPS) || defined(USE_MAG)` though - fixed.